bundles/core/src/test/resources/link/exporter-linkcomponent-v1.json

```diff
@@ -0,0 +1,11 @@
+{
+    "id": "link-00d7088753",
+    "linkText": "Link Component",
+    "linkTooltip": "Some Hover Tooltip Text",
+    "assetType": "ADAPTIVE_FORM",
```

Comment:
Is this naming aligned to the aem forms container component ? We should have an asset type enum common to all components

Reply:
The string representation of Asset Type aligns to what is available in aem forms container component, but internally enum used is different (i.e Link.AssetType.ADAPTIVE_FORM instead of AEMForm.FormType.ADAPTIVE_FORM).